### PR TITLE
Added Copenhagen meetups to meetup.yaml

### PR DESCRIPTION
**Pull request by jepp2078**

### CODE REVIEW

**Comment by jepp2078** [comment]
Seems this PR is running into the same issues as [#36](https://github.com/cloud-native-nordics/meetups/pull/36).

File: speakers.yaml [hunk 1]
```diff
@@ -74,3 +74,39 @@ speakers:
 - id: michael
   name: Michael Duergner
   company: zalando
+- id: ewelina
+  name: Ewelina Wilkosz
+  company: Praqma
```
Comment (kaspernissen) [inline speakers.yaml]: Can you reference the companies from the list here as well?

Comment (jepp2078) [reply speakers.yaml]: done

**Comment by kaspernissen** [comment]
Thank you @jepp2078

**Comment by Steiniche** [comment]
The make validate fails with:
```generator/bin/generator --validate
panic: runtime error: invalid memory address or nil pointer dereference
````

**Comment by kaspernissen** [comment]
@Steiniche yes, it's the same problem as the other PR: #39 with multiple speakers.

**Comment by kaspernissen** [comment]
As it turns out, it was a wrong reference that broke the build for this #36. I merged this PR #46 to provide better error handling. Rebase with master, and please try to generate again to get more information on where the problem is. @jepp2078

**Comment by kaspernissen** [comment]
Thanks @jepp2078. Can you rebase with master and resolve the conflicts - and this is ready to be merged? :)

**Comment by jepp2078** [comment]
@kaspernissen Rebase done. Build passes now.

**Pull request merged**